vendor/src/github.com/tent/canonical-json-go/encode.go

```diff
@@ -0,0 +1,620 @@
+// Copyright 2010 The Go Authors.  All rights reserved.
```

Comment:
Are we really going to depend on a fork on the standard `encoding/json` for a diff which I personally can't find because the file content was reordered?